Documentation/porting-imx.md

```diff
@@ -410,6 +410,20 @@ Edit `Dsdt-Sdhc.inc` and ensure that the SDHC instance on which your boot media
 ```c
 Package (2) { "Regulator1V8Exist", 0 }, // 1.8V Switching External Circuitry: Not implemented
 ```
+### PWM
+
+For best experience using the PWM WinRT APIs from UWP apps, some device properties may need to be set where some of which are generally required for UWP access to the PWM device interface while others are required for some PWM WinRT APIs to function correctly. More on these device interface properties can be found here in the [Setting device interface section](https://docs.microsoft.com/en-us/windows-hardware/drivers/spb/pulse-width-controller%20driver#setting-device-interface-properties) of the PWM DDI MSDN documentation.
+
+For example on setting the PWM deivce interface properties statically from an inf file, see the [Virtual PWM driver sample here](https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Drivers).
+
+For example on how to read ACPI _DSD in general from within a WDF driver and assigning so you can assign the PWM device properties in runtime, see the [i.MX SDHC driver here](https://github.com/MHesham/imx-iotcore/tree/public_preview/driver/sd/imxusdhc).
```

Comment:
@christopherco Didn't we remove the _DSD dependency from the imxusdhc stack?

https://github.com/ms-iot/imx-edk2-platforms/blob/imx/Platform/SolidRun/HummingBoardEdge_iMX6Q_2GB/AcpiTables/Dsdt-Sdhc.asl

Reply:
We did, but the code that deals with ACPI _DSD and _DSM is still there in the usdhc code, but it is dormant at the moment. Should be enabled again once we fix the SD power issues. Other aspects of the _DSD/_DSM needs some rework to be standard compliant, but the acpiutil code is still a good reference.

Reply:
The problem with any sort of _DSD/_DSM we have previously implemented is that the definitions/behaviors are not formally documented in a specification (Github readmes or code comments do not count). Upstream edk2 requires references a public documents for these types of OS-UEFI interactions.